test/framework/src/main/java/org/elasticsearch/test/MockLogAppender.java

```diff
@@ -27,10 +27,10 @@ public class MockLogAppender extends AbstractAppender {
 
     private static final String COMMON_PREFIX = System.getProperty("es.logger.prefix", "org.elasticsearch.");
 
-    private List<LoggingExpectation> expectations;
+    private final List<LoggingExpectation> expectations;
 
-    public MockLogAppender() throws IllegalAccessException {
-        super("mock", RegexFilter.createFilter(".*(\n.*)*", new String[0], false, null, null), null, false);
```

Comment:
are we able to parse multiline messages now? I wonder if this was the purpose of that filter.
This is used in testing only anyway, so if build is green I guess we can try this fix

Reply:
That was my thinking - if the builds all pass with this removed, then it's ok to remove it...